trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/category_pick/CategoryPickScreen.kt

```diff
@@ -42,7 +42,10 @@ internal fun CategoryPickScreen(
     ObserveAsEffect(effects = viewModel.effect) { effect ->
         when (effect) {
             is CategoryPickScreenEffect.NavigateBack -> navController.popBackStack()
-            is CategoryPickScreenEffect.NavigateToTrends -> navController.navigate(Route.Trends)
+            is CategoryPickScreenEffect.NavigateToHome -> navController.navigate(Route.ReelHome){
+                popUpTo(Route.Categories){ inclusive = true}
```

Comment:
Formatting is a bit cramped, it would be a good idea to utilize `ctrl + alt + L` here.